src/plugin-slots/DesktopUserMenuToggleSlot/README.md

```diff
@@ -0,0 +1,77 @@
+# Learning User Menu Toggle Slot
+
+### Slot ID: `org.openedx.frontend.layout.header_desktop_user_menu_toggle.v1`
+
+### Slot ID Aliases
+* `desktop_user_menu_toggle_slot`
+
+## Description
+
+This slot is used to replace/modify/hide the contents of the user menu toggle button on desktop sized screens.
+
+## Examples
+
+### Modify Username Text
+
+The following `env.config.jsx` will modify the 'username' text to something more generic:
+
+![Screenshot of modified label](./images/desktop_user_menu_modified_toggle.png)
+
+```jsx
+import { PLUGIN_OPERATIONS } from '@openedx/frontend-plugin-framework';
+import { faHouse } from '@fortawesome/free-solid-svg-icons';
+
+const modifyUserMenuToggle = ( widget ) => {
+  widget.content.username = "My Account";
```

Comment:
The screenshot says "My Profile" and this text says "My Account," it'd be good to make those match.